plugins/delegateAnnouncer/hooks/useDelegateAnnouncements.tsx

```diff
@@ -17,16 +17,18 @@ export function useDelegateAnnouncements(
 
     useEffect(() => {
         setIsLoading(true)
+        console.log('DAO', daoAddress)
             
         publicClient.getLogs({
             address: delegationContract,
             event: AnnounceDelegationEvent,
             args: {
                 dao: daoAddress
             } as any,
-            fromBlock: BigInt(53386422),
+            fromBlock: BigInt(10541166),
```

Comment:
Why not a computed value?

Reply:
My question is... Which one? Potentially we can query when was that contract deployed?

Reply:
Something that doesn't need to be searched for within the codebase itself. 
Could be a constant at the top, could be `now - 2 months`, I'm not sure

Reply:
Okey, moving it to env vars. Parsing from timestamp to blockNumber is a pain in the ass not worth fighting (specially because of the different chain block times...)